src/injected/utils/helpers.js

```diff
@@ -48,6 +54,20 @@ export const isArray = obj => (
   obj && typeof obj.length === 'number' && typeof obj.splice === 'function'
 );
 
+export const VM_KEY = '__Violentmonkey';
+export const isContentScript = !!window.browser
```

Comment:
I think it's more reliable to check with `bridge.mode`. Just that some more work is needed to access to `bridge`.
```js
const isContentScript = bridge.mode === INJECT_CONTENT;
```

Reply:
If we are going to support userscripts that override `Array#toJSON`, this check won't be needed.

Reply:
Uhm, isn't `helpers.js` also used in the main content script? That is in addition to the sandbox. I don't have a firm grasp, apparently on how bridge is used exactly, anyway.

Reply:
I probably should use `let` here and reassign to jsonStringifySafe if a script is received that wants to inject into the content script context.